src/main/java/com/theironyard/services/TwilioService.java

```diff
@@ -0,0 +1,62 @@
+package com.theironyard.services;
+
+import com.theironyard.entities.User;
+import com.theironyard.repositories.UserRepository;
+import com.twilio.Twilio;
+import com.twilio.rest.api.v2010.account.Message;
+import com.twilio.type.PhoneNumber;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+public class TwilioService {
+
+    @Autowired
+    UserRepository userRepo;
+
+    @Value("${twilio.sid}")
+    private String accountSid;
+
+    @Value("${twilio.token}")
+    private String authToken;
+
+    @Value("${twilio.phone}")
+    private String fromPhone;
+
+    @PostConstruct
+    public void init(){
+        Twilio.init(accountSid, authToken);
+    }
+
+    /* *
+     * On every hour, retrieves all users with alarmTime set to the current hour,
+     * loops through users and calls "sendSMS" with the user's phone number
+     * */
+    @Scheduled(cron = "* 00 * * * *")
```

Comment:
Are  you sure this works?  I believe you said to do it every second of midnight with this cron